build_srcs.bzl

```diff
@@ -321,3 +321,29 @@ MICROKERNEL_HDRS = [
     "src/xnnpack/vscaleextexp.h",
     "src/xnnpack/vunary.h",
 ]
+
+UKERNEL_CONFIG_SRCS = [
```

Comment:
Nit: I think this should be `MICROKERNEL_CONFIG_SRCS` for consistency with e.g. `MICROKERNEL_DEPS`, `MICROKERNEL_HDRS`.

Reply:
yea that makes sense.